docs/geedocs/5.3.7/answer/releaseNotes/relNotesGEE5_3_7.html

```diff
@@ -72,15 +72,11 @@ <h3>Quick search</h3>
 <h1>Release notes: Open GEE 5.3.7<a class="headerlink" href="#release-notes-open-gee-5-3-7" title="Permalink to this headline">¶</a></h1>
```

Comment:
We should only be commenting/evaluating the .rts files since the html is generated documentation that we also check in. It is also often outdated since the generator is often not ran for long periods of time outside of releases...